deployment/articles-dxp/01-saml/03-registering-a-service-provider.markdown

```diff
@@ -0,0 +1,118 @@
+---
+header-id: registering-a-saml-service-provider
+---
+
+# Registering a SAML Service Provider
+
+Setting up @product@ as a SAML Identity Provider is only useful if you
+can connect to one or more SAML Service Providers. Navigate to the Service
+Provider Connections tab of the SAML Admin Control Panel entry and click 
+the *Add Service Provider* button to add a SAML Service Provider.
+
+The New Service Provider page includes these options:
+
+**Name:** The name of the Service Provider with which to connect. The name can
+be anything; it's purely cosmetic.
+
+**Entity ID:** The Service Provider's entity ID. This value must match the
+entity ID declared in the Service Provider metadata.
+
+**Enabled:** Check this box to activate the Service Provider connection.
+
+**Assertion Lifetime:** Defines the number of seconds after which the SAML
+assertion issued by the Identity Provider should be considered expired.
+
+**Force Encryption:** If the SP does not provide a public key for encrypting the
+assertions, abort the single sign-on. 
+
+**Metadata:** Provide a URL to the Service Provider metadata XML file or
+manually upload the Service Provider metadata XML file. If you provide a URL,
+the XML file is retrieved and periodically polled for updates. The update
+interval can be configured in System Settings with the
+`saml.metadata.refresh.interval` property which specifies a number of seconds.
```

Comment:
Change: It's better to refer to the property name as it's displayed on the UI: `Runtime Metadata Refresh Interval ` (`com.liferay.saml.runtime.configuration.SamlConfiguration.getMetadataRefreshInterval()`)

Reply:
`saml.metadata.refresh.interval` was name of the property in 6.x prior to 3.1.0.

Reply:
Okay; I now have both. :-)

Reply:
@sez11a FYI this is true for all properties that haven't been removed. 
/cc @lipusz

Reply:
@stian-sigvartsen that's confusing to me, thanks for the heads-up.